templates/vue/src/components/lightbox/AccordionMedia.vue

```diff
@@ -184,7 +184,7 @@ button[disabled] {
 
 .media-container {
   height: 100%;
-  overflow: scroll;
+  overflow: hidden;
```

Comment:
This is not working for me. When my accordion is longer than the page, it doesn't let me scroll down to see the rest of the accordion.